src/scala/scripts/ScalaPBGenerator.scala

```diff
@@ -89,10 +92,21 @@ class ScalaPBGenerator extends Processor {
         }
         JarCreator.buildJar(Array(extractRequestResult.jarOutput, extractRequestResult.scalaPBOutput.toString))
     } finally {
+      deleteDir(tmpRoot)
       deleteDir(extractRequestResult.scalaPBOutput)
     }
   }
 
-  protected def exec(protoc: Path): Seq[String] => Int = (args: Seq[String]) =>
-    new ProcessBuilder(protoc.toString +: args: _*).inheritIO().start().waitFor()
+  protected def exec(protoc: Path): Seq[String] => Int = (args: Seq[String]) => {
+    val tmpFile = Files.createTempFile("stdout", "log")
+    try {
+      val ret = new ProcessBuilder(protoc.toString +: args: _*).redirectErrorStream(true).redirectOutput(ProcessBuilder.Redirect.to(tmpFile.toFile)).start().waitFor()
+      Try {
+        scala.io.Source.fromFile(tmpFile.toFile).getLines.foreach{System.out.println}
```

Comment:
this looks like debug. Can we remove?

Reply:
This is how we get the stdout/stderr from protoc into the bazel output stream. So can't remove. 

(prior to this PR any failure in protoc would result in just seeing a message that it exited with code 1 and no further info)

Reply:
oh yeah... I overlooked that. nit: can we do `foreach(System.out.println)` (use parens rather than `{}` when passing a method as a function argument)?